django/conf/global_settings.py

```diff
@@ -492,6 +492,7 @@
 PASSWORD_HASHERS = [
     'django.contrib.auth.hashers.PBKDF2PasswordHasher',
     'django.contrib.auth.hashers.PBKDF2SHA1PasswordHasher',
+    'django.contrib.auth.hashers.Argon2PasswordHasher',
```

Comment:
Making the hasher-selection smart enough to discard unusable hashers from the top of the list seems like a reasonable improvement, but out of scope for this PR. For the purposes of this PR, I think adding argon2 after PBKDF2 in the default list (as you have it now) is perfectly reasonable.